build_deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -exv
+
+BASE_IMG="rhel-fabric8-analytics-f8a-server-backbone"
+QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
+IMG="${BASE_IMG}:latest"
+
+GIT_HASH=`git rev-parse --short=7 HEAD`
+
+# build the image
+docker build  --no-cache \
+              --force-rm \
+              -t ${IMG}  \
+              -f ./Dockerfile.rhel .
+
+# Run tests
+./qa/runtests.sh
```

Comment:
Can you use the targets from Makefile which already does the image building?

https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/Makefile#L17

Reply:
We are planning to use app-sre org in quay.io for pushing these images. Also would like to confirm with @maorfr if there are any standards related to this on the App-SRE side.

Reply:
essentially the scripts belong to the development team and App SRE does not oversee them in any way. If the folks want to use the Makefile, I think we can do it in this PR. after all, we want to use this as an example for other services, so it would make sense to follow @arajkumar's suggestion. @aproddut does have a point in saying that we will be using a new image (quay.io/app-sre instead of quay.io/openshiftio).

Reply:
@maorfr , Okay.  then I leave it upto your decision. :) 
@aproddut If you feel like a dedicated script would be easy to maintain, go for it.

Reply:
I guess current jobs are using MakeFile for build and we are migrating this service in bits. With the quay repo location changing I guess we could use the script we created with this PR and then update it once the service is migrated to use the MakeFile for the build.
@arajkumar if you guys are fine with this approach can you resolve and merge this PR